hardhat.config.js

```diff
@@ -29,7 +29,8 @@ task("deploy", "Deploy a hedera contract")
       contractName: args.contract,
       // Optional, injected into the constructor, in this case for the "HelloWorld" Contract
       constructorParams: new ContractFunctionParameters()
-        .addAddress(new TokenId(0,0, 34111069).toSolidityAddress())
+        .addAddress(new TokenId(0,0, 34185686).toSolidityAddress())
```

Comment:
fix this or inject the token id through a env option ;-)

Reply:
Done it chief, updated .env.example to reflect changes.